src/encoded/tests/data/inserts/file_format.json

```diff
@@ -0,0 +1,130 @@
+        "uuid": "d13d06cf-218e-4f61-aaf0-91f226348b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "bam",
+        "standard_file_extension": "bam",
+        "allowed_extensions": ["bam"],
+        "description": "This format is used for aligned reads",
+        "uuid": "d13d06cf-218e-4f61-aaf0-91f226248b3c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "cool",
+        "standard_file_extension": "cool",
+        "allowed_extensions": ["cool"],
+        "description": "This format is used for contact matrices",
+        "uuid": "d13d06cf-218e-4f61-bbf0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "mcool",
+        "standard_file_extension": "mcool",
+        "allowed_extensions": ["mcool"],
+        "description": "This format is used for multi-resolution contact matrices",
+        "uuid": "d13d06cf-218e-4f61-ccf0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "other",
+        "standard_file_extension": "other",
+        "allowed_extensions": [""],
+        "description": "This format is used for other files",
+        "uuid": "d13d06cf-218e-4f61-11f0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "ome.tiff",
+        "standard_file_extension": "ome.tiff",
+        "allowed_extensions": ["ome.tiff"],
+        "description": "This format is used for Omero imaging files",
+        "uuid": "d13d06cf-218e-4f61-22f0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "tiff",
+        "standard_file_extension": "tiff",
+        "allowed_extensions": ["tiff"],
+        "description": "This format is used for Tiff image files",
+        "uuid": "d13d06cf-218e-4f61-33f0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "zip",
+        "standard_file_extension": "zip",
+        "allowed_extensions": ["zip"],
+        "description": "This format is used for zip compressed files",
+        "uuid": "d13d06cf-218e-4f61-44f0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "bwaIndex",
+        "standard_file_extension": "bwaIndex.tgz",
+        "allowed_extensions": ["bwaIndex.tgz"],
+        "description": "This format is used for bwaIndex files",
+        "uuid": "d13d06cf-218e-4f61-55f0-91f226248b2c",
+        "lab": "4dn-dcic-lab",
+        "award": "1U01CA200059-01"
+    },
+    {
+        "file_format": "compressed_fasta",
+        "standard_file_extension": "fasta.gz",
+        "allowed_extensions": ["fasta.gz", "fsa.gz"],
```

Comment:
I'd suggest adding ".fa.gz" to this list, since .fa is sometimes used as a fasta file extension.